### PR TITLE
Feature/rad id

### PR DESCRIPTION
**Pull request by LindsayYoung**
This adds another ID for RAD employees, as requested by @jwchumley

### CODE REVIEW

File: data/sql_updates/rad_analyst.sql [hunk 1]
```diff
@@ -16,6 +16,7 @@ select
     -- I am not sure we want to filter on this, if the name is different here it will behave differently than the other committee searches by name
     cmte_nm as committee_name,
     anlyst_id as analyst_id,
+    cast(anlyst_short_id as numeric) as analyst_short_id,
```
Comment (jontours) [inline data/sql_updates/rad_analyst.sql]: Haha.  Lovin' it!

Comment (jontours) [inline data/sql_updates/rad_analyst.sql]: On serious note, is there a reason this was needed to cast to numeric, noted that it is type numeric(4,0) in the view.

Comment (LindsayYoung) [inline data/sql_updates/rad_analyst.sql]: Both IDs are numbers and the other ID is numeric typed, so I thought the display of numeric ids should be consistent.

**Comment by jontours** [comment]
Tested filtering while pointing to dev, looks good!

**Pull request merged**